src/dao/api.dao.ts

```diff
@@ -247,70 +247,94 @@ export class NodeDao {
   };
 
   getFeedEntries = async (offset: number, limit: number, search: string) => {
```

Comment:
wait wait wait is this a general getFeedEntries used by LinkedTrust?

I was making changes to this also!

please make a separate endpoint for Candid so you can work quickly without breaking LinkedTrust

we can see later if we can refactor them together.  please keep separate for now

Reply:
wow this is not should be here lemme find out who made these changes because I didn't